rotary_irq_rp2.py

```diff
@@ -0,0 +1,67 @@
+# The MIT License (MIT)
+# Copyright (c) 2020 Mike Teachman
```

Comment:
Make sure to add your name to get credit for the contribution !
e.g.  
```
# Copyright (c) 2020 Mike Teachman
# Copyright (c) 2021 Eric Moyer
```

Reply:
Will do; thanks!